src/components/LoansByCategory/HelpmeChoose/HelpmeChooseTrigger.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+	<div
+		class="tw-w-full tw-flex tw-flex-col tw-bg-primary-inverse tw-rounded tw-p-1 lg:tw-p-2 tw-cursor-pointer"
+		@click="$emit('click', variant)"
+	>
+		<img :src="imgSource" class="tw-hidden lg:tw-block tw-w-3">
+		<h3 class="tw-text-h3 tw-text-primary-inverse lg:tw-mt-5 tw-text-center lg:tw-text-left">
+			{{ title }}
+		</h3>
+	</div>
+</template>
+
+<script>
+import progressIcon from '@/assets/images/helpmechoose/progress.svg';
+import kivaIcon from '@/assets/images/helpmechoose/kiva_mark.svg';
+import appleIcon from '@/assets/images/helpmechoose/apple.svg';
+
+export default {
+	name: 'HelpmeChooseTrigger',
+	props: {
+		variant: {
+			type: String,
+			default: ''
+		}
+	},
+	data() {
+		return {
+			progressIcon,
+			kivaIcon,
+			appleIcon
+		};
+	},
+	computed: {
+		title() {
+			let str = '';
+			if (this.variant === 'amountLeft') {
```

Comment:
Doesn't need to change, but these two methods with stacked if statements would typically be solved with switch statements instead.

Reply:
Nice suggestion, even we can update this removing str definition and just return the value for every case, but good to go for the experiment. Thanks @dyersituations !